### PR TITLE
docs: slim down install options

### PR DESCRIPTION
**Pull request by markmichon**
## Description
Based on feedback, this simplifies the install instructions when getting started to only include the script and homebrew. A new section with additional choices has been added before the FAQ section of the readme.

## Checklist

- [ ] I've added test coverage that shows my fix or feature works as expected.
- [ ] I've updated or added documentation if required.
- [ ] I've included usage information in the description if CLI behavior was updated or added.
- [x] PR title follows [Conventional Commits](https://www.conventionalcommits.org/) format

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -142,6 +115,30 @@ LOW: 0
...
 
+## :gear: Additional installation options
+
+### Configurable install script
+
+The default installation script works well for most use cases, but if you need more control you can customize the options by passing additional parameters.
+
+- `-b`: sets the installation directory (defaults to `./bin`)
+- `-d`: enables debug logging
+- `[tag]`: specifies a tag release (defaults to the latest release)
```
Comment (markmichon) [inline README.md:126]: @cfabianski I'm not sure on the specifics for customizing the install script but I've included the more obvious flags. Not sure about the `--` that is used in the example though?

Comment (cfabianski) [reply README.md:126]: That’s correct (including the —)

File: README.md [hunk 2]
```diff
@@ -142,6 +115,30 @@ LOW: 0
...
+```bash
+curl -sfL https://raw.githubusercontent.com/Bearer/curio/main/contrib/install.sh | sh -s -- -b /usr/local/bin
+```
+
+### Binary
+
+Download the archive file for your operating system/architecture from [here](https://github.com/Bearer/curio/releases/latest/). Unpack the archive, and put the binary somewhere in your $PATH (on UNIX-y systems, /usr/local/bin or the like). Make sure it has permission to execute:
+
+```bash
+chmod +x ./curio
```
Comment (cfabianski) [inline README.md]: Not sure that’s needed

File: docs/quickstart.md [hunk 3]
```diff
@@ -9,45 +9,16 @@ Discover data security risks and vulnerabilities in only a few minutes. In this
...
-curl -sfL https://raw.githubusercontent.com/Bearer/curio/main/contrib/install.sh | sh -s -- -b /usr/local/bin
-```
-
-### Binary
-
-Download the archive file for your operating system/architecture from [here](https://github.com/Bearer/curio/releases/latest/). Unpack the archive, and put the binary somewhere in your $PATH (on UNIX-y systems, /usr/local/bin or the like). Make sure it has permission to execute:
-
-```bash
-chmod +x ./curio
+brew install Bearer/curio/curio
```
Comment (cfabianski) [inline docs/quickstart.md:21]: Link to the repo github.com/bearer/homebrew-curio ?

**Review by gmontard — APPROVED** [review]

**Pull request merged**